DSC/dsc.py

```diff
@@ -44,21 +44,24 @@
 ExtensionShortName = 'DSCForLinux'
 DownloadDirectory = 'download'
 
-omi_package_prefix = 'packages/omi-1.1.0.ssl_'
-dsc_package_prefix = 'packages/dsc-1.1.1-294.ssl_'
+omi_package_prefix = 'packages/omi-1.4.2-5.ssl_'
+dsc_package_prefix = 'packages/dsc-1.1.1-926.ssl_'
```

Comment:
When do we plan to generate new DSC engine from master  and include as part of extension

Reply:
We can plan it in future releases while adding support for Python 3 dependent distros.